popup.html

```diff
@@ -585,86 +561,64 @@
-            <input required type="number" min="0" max="99" class="form-control input-sm" id="organizationID" name="organizationID"/>
-            <input type="number" min="0" max="99" class="form-control input-sm" id="suborgID" name="suborgID"/>
-            <input type="number" class="form-control input-sm go-right" id="subprimerID" name="subprimerID"/>
-          </div>
-          <div class="container">
-            <div class="quick-col form-group">
-              <label for="ftpID">
-                FTP
-              </label>
-              <br>
-              <input type="checkbox" class="checkbox" id="ftpID" name="ftpID" value="Yes"/>
+        </div>
+        <div class="form-field -required">
+          <label for="name">Your Name</label>
+          <input id="name" required>
+        </div>
+        <div class="form-field -required">
+          <label for="email">Your Email</label>
+          <input id="email" type="email" required>
+        </div>
+        <div class="form-field -required">
+          <label for="eventName">Event Name</label>
+          <input id="eventName" required>
+        </div>
+        <div class="form-field multi -required">
+          <label for="agencyID">Agency Office</label>
+          <input id="agencyID" type="number" min="0" max="99" required>
+          <input id="subAgencyID" type="number" min="0" max="99" required>
+          <input id="organizationID" type="number" min="0" max="99" required>
+          <input id="suborgID" type="number" min="0" max="99">
+        </div>
+        <div class="form-field multi">
+          <label for="subprimerID">Subprimer</label>
+          <input id="subprimerID" type="number">
+          <span class="_filler"></span>
+          <span class="_filler"></span>
+          <span class="_filler"></span>
+        </div>
+        <div class="form-field">
+          <label>Nature</label>
+          <fieldset>
+            <div class="_checkbox-wrapper">
+              <input type="checkbox" id="ftpID"> <label for="ftpID">FTP</label>
             </div>
-            <div class="quick-col form-group">
-              <label for="difficultyID">
-                Many Files
-              </label>
-              <br>
-              <input type="checkbox" class="checkbox" id="difficultyID" name="difficultyID" value="Yes"/>
+            <div class="_checkbox-wrapper">
+              <input type="checkbox" id="difficultyID"> <label for="difficultyID">Many Files</label>
             </div>
-            <div class="quick-col form-group">
-              <label for="databaseID">
-                Database
-              </label>
-              <br>
-              <input type="checkbox" class="checkbox" id="databaseID" name="databaseID" value="Yes"/>
+          </fieldset>
+          <fieldset>
+            <div class="_checkbox-wrapper">
+              <input type="checkbox" id="databaseID"> <label for="databaseID">Database</label>
```

Comment:
OK. Did it in an intentionally roundabout way for decoupling of collected data from sheet representation.